### ISSUE TITLE
Parallelization of Pysar

### ISSUE THREAD

**Issue by falkamelung** [geodesymiami/rsmas_insar#24]
It is only one step that needs to be parallelized. The first thing is to use all cores on one node.

talk to david and document where we stand on GitHub

**Comment by Ovec8hkin** [comment]
I will not be getting to this until semester break at the very earliest. I have attempted to test some basic Dask stuff on pegasus and it does not work as anticipated, which is due to the fact that we don't know how to configure the job or the code to make use of the multiple pegasus cores. 

I suggest you have someone from CCS or someone who has successfully used Dask on pegasus demonstrate and provide code examples and job submission files of working paralyzed jobs so we can emulate those.

I have expressed before my thoughts that this is not going to be as easy as we think, as evidenced by the fact that you have attempted to get this done for nearly 6 months and 0 progress has been made on it.
